src/main/resources/config/balancespec.yml

```diff
@@ -0,0 +1,14 @@
+---
+measures:
+    - key: UNIT
+      name: { en: " " }
+      isInteger: true
+types:
+    - key: BALANCE
+      name: { en: "Balance" }
+      measureKey: UNIT
+      isAlwaysShow: true
+      isWithPockets: true
+      removeZeroPockets: true
+      entityTypeKey:
+        - WALLET
```

Comment:
Maybe ACCOUNT?

Reply:
in exists tenant in can make error, when we try to get not exists balance from account ACCOUNT, and add not expecting component, but wallet, for my point of view it's correct place for balance